src/components/calculator.js

```diff
@@ -3,6 +3,7 @@ import './calculator.css';
 import CalculatorKey from './calculatorkeys';
 import calculate from '../logic/calculate';
 import DisplayResult from './display';
+import DisplayQuote from './DisplayQuote';
```

Comment:
- [ ] Based on project requirements, you're meant to call your `DisplayQuote` component directly into the main app component just as the calculator is stand alone. 
- This is asked to test your skills in using component standalone across your application, which is a major beauty of react.

## Requirement:
![image](https://user-images.githubusercontent.com/50990925/235031571-60e91008-010a-4943-8979-3b70e6287a1c.png)

Reply:
Ok @Shedrack-Sunday